com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -608,18 +608,20 @@ private MethodReference GetFastBufferWriterWriteMethod(string name, TypeReferenc
                             foreach (var constraint in method.GenericParameters[0].Constraints)
                             {
 #if CECIL_CONSTRAINTS_ARE_TYPE_REFERENCES
-                                var resolvedConstraint = constraint.Resolve();
+                                var resolvedConstraint = constraint.Resolve().DoSomething();
```

Comment:
`DoSomething();` ?

Reply:
oops... threw that in trying to verify which path was getting executed by forcing a compile error, forgot to take it out.